src/libostree/ostree-repo.h

```diff
@@ -107,6 +107,47 @@ OstreeRepo *  ostree_repo_create_at (int             dfd,
 
 #ifdef OSTREE_ENABLE_EXPERIMENTAL_API
 
+/**
+ * OstreeRepoLockType:
+ * The type of repository lock to acquire.
```

Comment:
Conventionally, this description line goes below the `@` lines. i.e.:
```
/**
 * OstreeRepoLockType:
 * @OSTREE_REPO_…
 * @OSTREE_REPO_…
 *
 * The type of repository lock to acquire.
 *
 * Since: 2017.14
 */
```

Reply:
Good call.